gui/qt/main_window.py

```diff
@@ -1966,7 +1966,8 @@ def update_console(self):
         c = commands.Commands(self.config, self.wallet, self.network, lambda: self.console.set_json(True))
         methods = {}
         def mkfunc(f, method):
-            return lambda *args: f(method, args, self.password_dialog)
+            return lambda *args, **kwargs: f(method, *args, password_getter=self.password_dialog,
+                                             **kwargs)
         for m in dir(c):
             if m[0]=='_' or m in ['network','wallet']: continue
```

Comment:
I wonder if this should read:

`if m[0]=='_' or m in ['network','wallet','config']: continue`

Because Commands has self.config as an attribute...

Reply:
I'm pretty sure it was a bug and it should.  Would you mind adding a commit that does that?

Reply:
If I understand you correctly, that isn't related to any of my changes, so I think it would be better done by someone else who can build the desktop app and test it for themselves.

Reply:
Right right.. just figured you'd throw it into this big ball o' commits. No worries -- I just committed the fix. 👍